### PR TITLE
Update travis config

### PR DESCRIPTION
**Pull request by Xerkus**
- Removed docs build step
- Removed php 5.5 from build matrix
- Removed irc and slack notifications
- Removed fast_finish flag

Closes #20

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -48,49 +35,49 @@ matrix:
...
+  - if [[ $CS_CHECK == 'true' ]]; then composer license-check ; fi
 
 after_script:
-  - if [[ $TEST_COVERAGE == 'true' ]]; then travis_retry composer upload-coverage ; fi
+  - if [[ $TEST_COVERAGE == 'true' ]]; then composer upload-coverage ; fi
 
-after_success:
-  - if [[ $DEPLOY_DOCS == "true" ]]; then ./zf-mkdoc-theme/deploy.sh ; fi
+notifications:
+email: false
```
Comment (webimpress) [inline .travis.yml]: We need here two spaces indent

Comment (Xerkus) [reply .travis.yml]: good catch

File: .travis.yml [hunk 2]
```diff
@@ -48,49 +35,49 @@ matrix:
...
-  - composer show --installed
+  - if [[ $TEST_COVERAGE == 'true' ]]; then travis_retry composer require --dev $COMPOSER_ARGS $COVERAGE_DEPS ; fi
+  - stty cols 120 && composer show
 
 script:
   - if [[ $TEST_COVERAGE == 'true' ]]; then composer test-coverage ; else composer test ; fi
-  - if [[ $CHECK_CS == 'true' ]]; then composer cs-check ; fi
-  - if [[ $DEPLOY_DOCS == "true" && "$TRAVIS_TEST_RESULT" == "0" ]]; then travis_retry curl -sSL https://raw.githubusercontent.com/zendframework/zf-mkdoc-theme/master/theme-installer.sh | bash ; fi
+  - if [[ $CS_CHECK == 'true' ]]; then composer cs-check ; fi
+  - if [[ $CS_CHECK == 'true' ]]; then composer license-check ; fi
```
Comment (webimpress) [inline .travis.yml]: We don't have specified this script in composer.json

**Comment by froschdesign** [comment]
@Xerkus 
Maybe this template could help: https://github.com/zendframework/maintainers/blob/master/template/.travis.yml

**Comment by webimpress** [comment]
@froschdesign Template is for libraries which supports only PHP ^7.1, not sure if we want here bump PH requirements to PHP ^7.1 and drop PHP 5.5/5.6/7.0/hhvm?

**Comment by Xerkus** [comment]
i've used zend-expressive as template as it supports php 5.6
Going to add docheader in a bit

**Comment by webimpress** [comment]
@Xerkus
> Going to add docheader in a bit

We are not using it in new libraries any more. For these license header checks we are going to use zend-coding-standard in future... There is already PR: https://github.com/zendframework/zend-coding-standard/pull/2

**Comment by Xerkus** [comment]
@webimpress good to know, thanks

**Pull request merged**